client/src/index.test.ts

```diff
@@ -25,7 +26,7 @@ describe('request', () => {
       };
     });
 
-    users.request('GET', '/foo');
+    await users.request('GET', '/foo');
```

Comment:
I'd like some kind of eslint rule to avoid this go unnoticed (ie: https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/no-floating-promises.md)

Reply:
Agreed. This would be great. In the frontend code as well. I'll make a note and create an issue soon.